src/bin/printing.rs

```diff
@@ -0,0 +1,99 @@
+//! # Printing
+//!
+//! This sample reads text from two Entry fields,
+//! shows a print dialog and prints both texts one below
+//! the other.
+
+extern crate cairo;
+extern crate gio;
+extern crate gtk;
+extern crate pango;
+extern crate pangocairo;
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use std::env::args;
+
+fn print(window: &gtk::Window, value1: String, value2: String) {
+    let print_operation = gtk::PrintOperation::new();
+
+    // Currently unused
+    // Could be used to check whether there was a success in printing
+    //let print_operation_result: gtk::PrintOperationResult;
```

Comment:
How would it be used?

Reply:
It seems from the docs that it's the return of the `PrintOperationExt` `run()` method that starts the printing, and it returns one of these:
```
Error
Apply
Cancel
InProgress
```

Reply:
I 'm now printing it with the println! macro.